Services/NewsService/Controllers/NewsPublisherController.cs

```diff
@@ -11,10 +11,15 @@ namespace NewsService.Controllers
     /// To verify NewsService work in the url - field add "/NewsService/get?newsPublisherType=CentralBank"
     /// You can select only NewsService to start
     /// </summary>
+    
     public class NewsServiceController : ControllerBase
     {
         [Route("[controller]/get")]
         [HttpGet]
+
+        ///<summary>
+        /// Return news depends on a publisher type
+        /// </summary>
         public String GetNews([FromQuery] NewsPublisherTypes newsPublisherType)
         {
             INewsPublisher newsPublisher = NewsPublisherFactory.CreateNewsPublisher(newsPublisherType);
```

Comment:
Not sure we need that name for Create method, we already have NewsPublisherFactory as parent object, for what this duplication?